libfreerdp/crypto/base64.c

```diff
@@ -24,8 +24,9 @@
 #include <freerdp/crypto/crypto.h>
 
 static const char base64[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+static const char base64url[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_";
```

Comment:
I'm fine with the changes could you add some tests in `libfreerdp/crypto/test/TestBase64.c` for these new functions ?